static/src/css/vendor/openlayers.scss

```diff
@@ -0,0 +1,45 @@
+@use '../utils';
+
+.ol-attribution {
+  font-family: Avenir Next W00, Helvetica Neue, Helvetica, Arial, sans-serif;
```

Comment:
Didnt catch this before. We should make this use `$font-family-sans-serif` from bootstrap. Should be something like this:

```
@use 'vendor/bootstrap/configuration' as bootstrap;
...
font-family: boostrap.$font-family-sans-serif;
font-size: boostrap.$font-size-base;
...
```